bot/exts/fun/off_topic_names.py

```diff
@@ -3,9 +3,11 @@
 import difflib
```

Comment:
Can you rebase onto main ? 
There are breaking changes since the `get_or_fetch_channel` isn't available anymore, you need to import it from `pydis_core`

Reply:
Fixed in e296bb56